src/docs/product/relay/options.mdx

```diff
@@ -62,13 +62,13 @@ These settings fine tune caching of project states.
 
 ## Spooling
 
-| key                             | type    | value            | description                                                                                                                  |
-| ------------------------------- | ------- | ---------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| spool.envelopes.path            | String  | default: `null`  | The path to the spool file used to save envelopes to disk. If provided, enables on-disk spooling.                            |
-| spool.envelopes.max_memory_size | String  | default: `500MB` | The maximum in-memory buffer for incoming envelopes. The buffer will be spooled to disk when the configured size is reached. |
-| spool.envelopes.max_disk_size   | String  | default: `500MB` | The maximum on-disk spool size.                                                                                              |
-| spool.envelopes.max_connections | Integer | default: `20`    | The maximum number of connections in the pool to manage on-disk spooling.                                                    |
-| spool.envelopes.min_connections | Integer | default: `10`    | The minimum number of connections in the pool to manage on-disk spooling.                                                    |
+| key                             | type    | value            | description                                                                                                                                                                                                             |
+| ------------------------------- | ------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| spool.envelopes.path            | String  | default: `null`  | The path to the spool file used to save envelopes to disk. If provided, Relay will create the directory structure and enable on-disk spooling. Note that `~` is interpreted as a literal and not as the home directory. |
```

Comment:
```suggestion
| spool.envelopes.path            | String  | default: `null`  | The path to the spool file used to save envelopes to disk. If provided, enables on-disk spooling. Relay will create the directory structure and enable on-disk spooling. Note that `~` is interpreted as a literal and not as the home directory. |
```

Reply:
Reworded to something similar, let me know what you think. I don't like duplicating the "enable on-disk" part.